<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-C8MG2FQ" name="Test" tests="3" failures="1" timestamp="31 May 2021 03:53:51 GMT" time="9.827" errors="0">
  <testcase name="fetch_CountriesList" time="6.725" classname="com.restassured.APIAutomation.CountriesandCapitals"/>
  <testcase name="fetch_Capitallist" time="1.455" classname="com.restassured.APIAutomation.CountriesandCapitals"/>
  <testcase name="Negative_TC_fetching_the_Capitaldetails" time="1.607" classname="com.restassured.APIAutomation.CountriesandCapitals">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
	at com.restassured.APIAutomation.Assertions.VerifyStatusCode(Assertions.java:18)
	at com.restassured.APIAutomation.CountriesandCapitals.Negative_TC_fetching_the_Capitaldetails(CountriesandCapitals.java:340)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Negative_TC_fetching_the_Capitaldetails -->
</testsuite> <!-- Test -->
